components/scream/src/physics/p3/p3_functions_conservation.cpp

```diff
@@ -5,11 +5,11 @@ namespace scream {
 namespace p3 {
 
 /*
- * Explicit instatiation for doing p3 conservation functions on Reals using the
+ * Explicit instantiation for doing p3 conservation functions on Reals using the
```

Comment:
@PeterCaldwell  This typo is in most of the p3_function_*.cpp source files. Do we want to fix the typo everywhere?

Reply:
In general, I feel like typos in comments could make people reading scream code for the first time think we're sloppy and generally give them a bad feeling about our code... so yeah, I do think we should fix typos when we see them. We can do this during our 'make useful variables names' pass as well, so only fix things that are quick to find.

Reply:
@PeterCaldwell  Yeah, I'm in full agreement with your feelings on typos and I think they should be fixed now. I'll go ahead and fix it everywhere, I have been involved in some projects where bringing files into a PR for the singular purpose of fixing comment typos is frowned upon, so I wanted to ask before I acted.